engine-precompiles/src/xcc.rs

```diff
@@ -19,8 +19,14 @@ use evm_core::ExitError;
 pub mod costs {
     use crate::prelude::types::{EthGas, NearGas};
 
-    // TODO(#483): Determine the correct amount of gas
-    pub(super) const CROSS_CONTRACT_CALL: EthGas = EthGas::new(0);
+    /// Base EVM gas cost for calling this precompile.
+    pub const CROSS_CONTRACT_CALL_BASE: EthGas = EthGas::new(115_000);
+    /// Additional EVM gas cost per bytes of input given.
+    pub const CROSS_CONTRACT_CALL_BYTE: EthGas = EthGas::new(2);
+    /// EVM gas cost per NEAR gas attached to the created promise.
+    /// Derived from the gas report https://hackmd.io/@birchmd/Sy4piXQ29
```

Comment:
It would be helpful to give quite a bit of context of how the value of 175,000,000 was derived, and mention that it was rounded to keep it consistent.

Same goes for the 115_000 value above.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/564/commits/12f123862d704aa12336e46227918738c854487d